frontend/components/media/MediaMap.vue

```diff
@@ -167,7 +175,18 @@ onMounted(() => {
                 tiles: ["https://tile.openstreetmap.org/{z}/{x}/{y}.png"],
                 tileSize: 256,
                 attribution:
-                  '<a href="https://www.openstreetmap.org/about" target="_blank">Data &copy; OpenStreetMap contributors</a>',
+                  '<a href="https://www.openstreetmap.org/copyright" target="_blank">&copy; OpenStreetMap contributors</a>',
+              },
+              "cycle-raster-tiles": {
+                type: "raster",
+                tiles: [
+                  "https://a.tile-cyclosm.openstreetmap.fr/cyclosm-lite/{z}/{x}/{y}.png",
+                  "https://b.tile-cyclosm.openstreetmap.fr/cyclosm-lite/{z}/{x}/{y}.png",
+                  "https://c.tile-cyclosm.openstreetmap.fr/cyclosm-lite/{z}/{x}/{y}.png",
```

Comment:
Adding thoughts here..

It could potentially be good if we're able to load the tile source URLs from env vars - instead of configuring these hard-coded. Same as with `https://tile.openstreetmap.org` for the default layer source URL and the geocoding API URL `https://nominatim.openstreetmap.org`.

To note - as they are, the latter two are probably easy to do as we can simply replace them with an env var each. What about multiple-subdomain URLs though as in here for CyclOSM? We might have to configure a way to  be able to add more or less env vars if the number of subdomains change.

Or - we decide to not concern ourselves too much given that the mentioned services are likely to be and continue fairly stable in the near future? :thinking::shrug:

Reply:
Ya I'd say for now let's not concern ourselves too much with this, @wkyoshida :) We can revisit it later when we have a bit more capacity 😊